sci-libs/ignition-math/ignition-math-6.15.1.ebuild

```diff
@@ -9,17 +9,13 @@ inherit cmake vcs-snapshot
 DESCRIPTION="A small, fast, and high performance math library for robot applications"
 HOMEPAGE="https://github.com/ignitionrobotics/ign-math"
 SRC_URI="https://github.com/ignitionrobotics/ign-math/archive/${PN}6_${PV}.tar.gz"
+S="${WORKDIR}/${PN}6_${PV}"
 
 LICENSE="Apache-2.0"
 SLOT="6/6"
 KEYWORDS="~amd64"
-IUSE=""
 
-RDEPEND=""
-DEPEND="${RDEPEND}
+BDEPEND="dev-build/ignition-cmake:2"
+DEPEND="${BDEPEND}
```

Comment:
I'd say this is also a style difference. `DEPEND="${BDEPEND}" rarely makes sense so it may be easier to grasp the real deps by writing them plainly, as they were.